ext/afform/core/CRM/Afform/Page/AfformBase.php

```diff
@@ -30,7 +30,19 @@ public function run() {
     if (!empty($afform['title'])) {
       $title = strip_tags($afform['title']);
       CRM_Utils_System::setTitle($title);
-      CRM_Utils_System::appendBreadCrumb([['title' => $title, 'url' => CRM_Utils_System::url(implode('/', $pagePath), NULL, FALSE, '!/')]]);
+    }
+
+    // If the user has "access civicrm" append home breadcrumb
+    if (CRM_Core_Permission::check('access CiviCRM')) {
+      CRM_Utils_System::appendBreadCrumb([['title' => ts('CiviCRM'), 'url' => CRM_Utils_System::url('civicrm')]]);
+      // If the user has "admin civicrm" & the admin extension is enabled
+      if (CRM_Core_Permission::check('administer CiviCRM') && CRM_Utils_Array::findAll(
+          \CRM_Extension_System::singleton()->getMapper()->getActiveModuleFiles(),
+          ['fullName' => 'org.civicrm.afform_admin']
+        )) {
```

Comment:
Would `getMapper()->isActiveModule('afform_admin')` be simpler?

Reply:
Yes thanks @totten. Didn't know about that function.